docs/pages/use_cases/COMPSs.md

```diff
@@ -0,0 +1,62 @@
+---
+page_id: COMPSs
+title: COMPSs Workflow Provenance
+description: COMPSs recording of Workflow Provenance for governance and reproducibility of computational experiments
+url: https://compss.bsc.es/
+image: COMPSs.png
+domains: [informatics] # should match item(s) in pages/domains
+tasks: [conduct_research, curation, data_handling, software_development] # should match item(s) in pages/tasks
+roles: [information_architect, researcher, software_developer] # should match item(s) in pages/roles
+---
+<!--
+   Copyright 2019-2022 RO-Crate contributors
+   <https://github.com/ResearchObject/ro-crate/graphs/contributors>
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+
+# COMPSs
```

Comment:
Ideally this and the `title` in the front matter should match

Reply:
You can change the title to "COMPSs" then, thanks!!!!

Reply:
Shall I do the fixes?